web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -77,34 +73,35 @@ const LicenseStatusLookup: Record<LicenseStatus, string> = {
   REVOKED: Config.licenseSearchTask.revokedStatusText,
 };
 
+const pendingStatuses: Set<LicenseStatus> = new Set([
+  "PENDING",
+  "DRAFT",
+  "SUBMITTED",
+  "UNDER_INTERNAL_REVIEW",
+  "SPECIAL_REVIEW",
+  "PENDING_DEFICIENCIES",
+  "DEFICIENCIES_SUBMITTED",
+  "CHECKLIST_COMPLETED",
+  "APPROVED",
+  "PENDING_RENEWAL",
+  "PENDING_REINSTATEMENT",
+]);
+
+const isPending = (licenseStatus: LicenseStatus): boolean => {
+  return pendingStatuses.has(licenseStatus);
+};
+
 export const LicenseStatusReceipt = (props: Props): ReactElement => {
-  const [theme, setTheme] = useState<PermitTheme>(pendingPermitTheme);
+  const [permitColorScheme, setPermitColorTheme] = useState<PermitTheme>(pendingPermitTheme);
   const { business } = useUserData();
 
-  const isPending = (licenseStatus: LicenseStatus): boolean => {
-    const pendingStatus: LicenseStatus[] = [
-      "PENDING",
-      "DRAFT",
-      "SUBMITTED",
-      "UNDER_INTERNAL_REVIEW",
-      "SPECIAL_REVIEW",
-      "PENDING_DEFICIENCIES",
-      "DEFICIENCIES_SUBMITTED",
-      "CHECKLIST_COMPLETED",
-      "APPROVED",
-      "PENDING_RENEWAL",
-      "PENDING_REINSTATEMENT",
-    ];
-    return pendingStatus.includes(licenseStatus);
-  };
-
   useEffect(() => {
     if (props.status === "ACTIVE") {
-      setTheme(activePermitTheme);
+      setPermitColorTheme(activePermitTheme);
     } else if (isPending(props.status)) {
-      setTheme(pendingPermitTheme);
+      setPermitColorTheme(pendingPermitTheme);
     } else {
-      setTheme(grayPermitTheme);
+      setPermitColorTheme(inactivePermitTheme);
     }
   }, [props.status]);
```

Comment:
[pebble] Likewise, can this be accomplished outside a `useEffect()`?

Reply:
I don't think a useEffect is necessary